pkg/cache/unavailableofferings.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 const (
-	UnavailableOfferingsTTL = 3 * time.Minute
+	UnavailableOfferingsTTL             = 3 * time.Minute
+	UnavailableOfferingsCleanupInterval = 10 * time.Minute
```

Comment:
In other places we've used the same TTL for both. Any thoughts on this?

Reply:
I'm not sure what you mean. You're asking if we should use a minute for unavailable offerings in this case. Or are you asking if the cleanup interval should be the same as the TTL? I'm not sure I fully understand the effects/scope of reducing the cleanup interval for expired entries. I'd imagine it's just a memory/cpu burden tradeoff and more of a heuristic in general?